tests/regression/test_fdm.py

```diff
@@ -95,9 +94,50 @@ def test_p_independence(mesh, expected, variant):
         problem = LinearVariationalProblem(a, L, uh, bcs=bcs)
         solver = LinearVariationalSolver(problem, solver_parameters=fdmstar)
         solver.solve()
-        nits.append(solver.snes.ksp.getIterationNumber())
+        assert solver.snes.ksp.getIterationNumber() <= nits
```

Comment:
How come the number of expected iterations didn't change as you solved new PDEs? Is it a coincidence that the iteration counts are the same?